src/EventAccumulator.ts

```diff
@@ -0,0 +1,99 @@
+import type { CountlyWebSdk, IEventAccumulator, CountlyEvent, CountlyEventData } from 'countly-sdk-web'
+
+const eventDefaults: CountlyEventData = {
+  key: '',
+  count: 1,
+  sum: 1,
+  dur: Date.now(),
```

Comment:
duration should be an amount of time, not a timestamp. This duration is currently going to be the full time since 1970.

We should probably default this to `0` or even leave it unset entirely.

see 
- https://github.com/Countly/countly-sdk-web/blob/1c72a3b5d4c1e66031e3ecc5779adfe030ab21e0/lib/countly.js#L3501
- https://support.count.ly/hc/en-us/articles/360037441932-Web-analytics-JavaScript-#timed-events